src/Controllers/MyRadio/login.php

```diff
@@ -16,7 +16,7 @@
  */
 
 if (isset($_SESSION['memberid'])) {
-    URLUtils::redirect(Config::$base_url);
+    URLUtils::redirectWithMessage("/", "", "It seems you were trying to login while still being logged in, if that is not what happened, please contact the Computing Team.");
```

Comment:
Meh, it was mainly because otherwise you've just redirected to the homepage for an unknown reason. It hits this when the API is being weird (like now).